### PR TITLE
chore(ci): split environments/geo

### PR DESCRIPTION
**Pull request by wlee221**
#### Description of changes

This migrates `environments/geo` resources to the new account. This will go into a feature branch `split-environments/main`, which will be merged once all resources has been migrated.

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: environments/pull-environment.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ AWSCLOUDFORMATIONCONFIG="{\
 \"profileName\":\"$AWS_PROFILE\",\
 \"accessKeyId\":\"$AWS_ACCESS_KEY_ID\",\
 \"secretAccessKey\":\"$AWS_SECRET_ACCESS_KEY\",\
-\"region\":\"us-east-1\"\
+\"region\":\"us-east-2\"\
```
Comment (wlee221) [inline environments/pull-environment.sh:28]: New environments are now in `us-east-2`. We will incrementally migrate all e2e resources to it, starting with `geo` in this PR.

File: .github/workflows/poc-test-environments.yml [hunk 2]
```diff
@@ -30,4 +30,11 @@ jobs:
         with:
           node-version: lts/*
           cache: 'yarn'
-      # TODO: add yarn environments <category> pull incrementally
+      - name: Add Amplify CLI
+        run: yarn global add @aws-amplify/cli
+
+      - name: Pull down Geo AWS environments, on cache miss
```
Comment (wlee221) [inline .github/workflows/poc-test-environments.yml:36]: This is only for POC purposes to demonstrate that `yarn environments geo pull` works. See "status checks" below.

File: environments/geo/basic-map/package.json [hunk 3]
```diff
@@ -3,6 +3,6 @@
   "name": "basic-map",
   "version": "0.0.1",
   "scripts": {
-    "pull": "amplify pull --appId dao8s4r01dxsy --envName staging"
+    "pull": "amplify pull --appId d1xll51xlvl8ri --envName staging"
```
Comment (0618) [inline environments/geo/basic-map/package.json:6]: where's this new app-id from?

File: environments/pull-environment.sh (hunk 1 above)
Comment (0618) [reply environments/pull-environment.sh:28]: Do we have any documentation which account and region contents what? Might be helpful to refer in the PR description?

**Review by 0618 — APPROVED** [review]

**Review by ErikCH — APPROVED** [review]

File: environments/pull-environment.sh (hunk 1 above)
Comment (zchenwei) [reply environments/pull-environment.sh:28]: For my knowledge, why we put it in `CMH`?

**Review by zchenwei — APPROVED** [review]

File: environments/pull-environment.sh (hunk 1 above)
Comment (wlee221) [reply environments/pull-environment.sh:28]: @0618 I tried to keep internal details out of pr description, but I realize this was really unclear. I'll share that async.

File: environments/geo/basic-map/package.json (hunk 3 above)
Comment (wlee221) [reply environments/geo/basic-map/package.json:6]: Yes, this is from the new aws account.

File: environments/pull-environment.sh (hunk 1 above)
Comment (wlee221) [reply environments/pull-environment.sh:28]: @zchenwei No strong reason, but mainly to keep resources in the same region as other aws resources we own.

**Pull request merged**